pep/app/pods/components/document/text/component.ts

```diff
@@ -31,22 +67,47 @@ export default class DocumentText extends Component<DocumentTextArgs> {
     @service notifications!: NotificationService;
     @service intl!: IntlService;
     @service theme!: ThemeService;
+    @service router!: RouterService;
 
     @tracked xml?: XMLDocument;
     containerElement?: HTMLElement;
 
+    tippyOptions = {
+        theme: 'light',
+        allowHTML: true,
+        interactive: true,
+        trigger: 'mouseenter focus click',
+        onClickOutside(instance: Instance<Props>) {
+            instance.hide();
+        }
+    };
+
     constructor(owner: unknown, args: DocumentTextArgs) {
         super(owner, args);
-        this.parseDocumentText(args.text);
+        this.parseDocumentText(
+            args.document.accessLimited ? args.document.abstractCleaned : args.document.documentCleaned
```

Comment:
could we ditch these `*Cleaned` getters entirely now, since we are using the XML versions?

Reply:
Possibly, I'll make that change in 005 which will give me time to play with it and make sure nothing breaks